pages/docs/tutorials/kafka/managing-schemas-using-schema-registry.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Managing schemas using Schema Registry
+description: A tutorial teaching how to manage schemas using Schema Registry.
+weight: 250
+---
+
+## Introduction
+In the [previous tutorial](/docs/tutorials/kafka/configure-kafka-avro), you've learned how to add Avro Schemas to your AsyncAPI document. Now, you will learn how to save your schema in a central Schema Registry and reuse it from there.
```

Comment:
The first reference to Avro on a page should be "Apache Avro". Subsequent references to just "Avro" are fine.

It would be good to put something like this at the end of the tutorial

> Apache and Apache Avro are trademarks of The Apache Software Foundation.